pydatastructs/linear_data_structures/stack.py

```diff
@@ -0,0 +1,98 @@
+"""stack.py file containning defination of 'Stack' datastructure"""
+
+from __future__ import print_function, division
+from copy import deepcopy as dc
+
+from ..utils.type_utils import _check_type
+
+__all__ = ["Stack"]
```

Comment:
```python
__all__ =  [
    'Stack'
]
```

Reply:
I am using [black]() for code formatting, handy tool, any specific reason to use above format.

Reply:
We are following the coding style of `SymPy`, so please follow that.